app/src/main/java/com/example/bookwormadventuresdeluxe2/Notification.java

```diff
@@ -0,0 +1,22 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.widget.ImageView;
+
+import com.example.bookwormadventuresdeluxe2.Utilities.Status;
+
+public class Notification extends Book
```

Comment:
Just +1ing Kate and Richmond, leaving this comment so github marks this as changes requested explicitly (the other thread wasn't triggering the requested change status)